### PR TITLE
add support to fetch trained logistic regression model in the logreg metric

### LINKED ISSUE #58: add logistic regression models to metric state

### PR DESCRIPTION
**Pull request by MayankAgarwal**
fixes #58 

Adds a property to get trained Logistic regression models for a fitted Logistic Regression Sensitive Subspace metric.

```
metric = inFairness.distances.LogisticRegSensitiveSubspace()
models = metric.logistic_regression_models
```

The `models` variable will now contain a list of logistic regression models modeling `P(K|X)`

*Note:* cannot add this to the state of the metric since state_dict only supports tensors to be specified, whereas this is a sklearn object.

### CODE REVIEW

**Review by moonfolk — APPROVED** [review]
Perfect for streamlined demos/use-cases, but would be good to accommodate load functionality too.
